src/main/java/ch/sbb/polarion/extension/pdf/exporter/util/HtmlProcessor.java

```diff
@@ -1137,6 +1140,10 @@ public String replaceImagesAsBase64Encoded(String html) {
         return html;
     }
 
+    public String internalizeLinks(String html) {
```

Comment:
why do you need this method?

Reply:
The HtmlToPdfConverter and PdfConverter don't call HtmlLinksHelper directly, but only through the HtmlProcessor.internalizeLinks().
The reason is minimizing injected dependencies in HtmlToPdfConverter and PdfConverter and logically operation is html processing like replacing base64 encoded images (just too large to be implemented directly in HtmlProcessor)